tools/nimweb.nim

```diff
@@ -180,8 +179,6 @@ proc parseCmdLine(c: var TConfigData) =
         c.gitURL = val
       of "git.commit":
         c.nimArgs.add("--git.commit:" & val & " ")
-      of "git.devel":
-        c.nimArgs.add("--git.devel:" & val & " ")
```

Comment:
It should be a noop or deprecated message, otherwise it breaks external library.